src/tcgwars/logic/impl/gen8/ChillingReign.groovy

```diff
@@ -989,7 +989,7 @@ public enum ChillingReign implements LogicCardInfo {
             }
             onAttack {
               if (my.deck) {
-                my.deck.search(min:1, max: 2, "Select up to 2 cards", { true }).moveTo(hidden: true, my.hand)
+                my.deck.search(min:0, max: 2, "Select up to 2 cards", { true }).moveTo(hidden: true, my.hand)
```

Comment:
Attacks that "search for up to x _cards_" must take at least one, when a certain filter isn't specified

```suggestion
                my.deck.search(min:1, max: 2, "Select up to 2 cards", { true }).moveTo(hidden: true, my.hand)
```